src/framework/preferences/js/StarterSchemas.js

```diff
@@ -21,11 +21,41 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
      * contrast, table of contents, inputs larger and emphasize links
      *******************************************************************************/
 
+    var check;
+    var widthCheck = function() {
+        var width = $(window).width();
+        check = (width<640);
+        if (check) {
+            fluid.contextAware.makeChecks({
+                "fluid.responsiveCheck": true
+            });
+        }
+        else {
+            fluid.contextAware.forgetChecks("fluid.responsiveCheck");
+            fluid.defaults("fluid.prefs.auxSchema.starter.default", {
+                gradeNames: ["fluid.contextAware"],
+                contextAwareness: {
+                    sliderVariety: {
+                        checks: {
+                            jQueryUI: {
+                                contextValue: "{fluid.prefsWidgetType}",
+                                equals: "jQueryUI",
+                                gradeNames: ["fluid.prefs.auxSchema.starter.textSize.jQueryUI", "fluid.prefs.auxSchema.starter.lineSpace.jQueryUI", "fluid.prefs.auxSchema.starter.blueColorFilter.jQueryUI"]
```

Comment:
Instead of nesting contextAwarenesses, have you tried to narrow down the context checks into 4 scenarios:
- jQuery + non-mobile
- jQuery + mobile
- nativeHtml + non-mobile
- nativeHtml + mobile